### PR TITLE
docs: improve README + newline fixes

### PR DESCRIPTION
**Pull request by rubenfiszel**

### CODE REVIEW

**Comment by rubenfiszel** [comment]
@mrchief : would appreciate your feedback on this.

File: README.md [hunk 1]
```diff
@@ -1,68 +1,70 @@
...
+Appends changelog diff based on the conventional commits format history and bump a file containing, among others, that version as meta-information. Such file can be a package.json in the npm world, or a yaml file (file which contains contents either in json or yaml). The next version is inferred based on the changes since last released tag.
 
 ## Introduction
 
 This Github action takes the following assumptions:
 
 - The commit history of the master branch mostly respect the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/). The action will still makes sense if it is partially or even not respected all, but it will not be able to do the proper next semantic version inference as well as pretty printing the changes in the changelog.
-- You have a _manifest_ file which is either YAML or JSON that has version as one of its top field.
+- You have a _manifest_ file which is either YAML or JSON that has version as one of its top-level field.
 - You have a _changelog_ file in markdown in which you want to append
```
Comment (mrchief) [inline README.md]: Doesn't it generate a changelog if none exist? If so, I'd suggest the following:

```suggestion
- You have a _changelog_ file in markdown in which you want to append. If none exists, it generates `CHANGELOG.md` in your root folder.
```

File: README.md [hunk 2]
```diff
@@ -158,6 +165,49 @@ jobs:
...
+```yaml
+inputs:
+  changelog_file:
+    description: 'changelog file path'
+    required: false
+    default: './CHANGELOG.md'
+  manifest_file:
+    description: 'manifest file path'
+    required: false
+    default: './manifest.yml'
```
Comment (mrchief) [inline README.md]: How about 

```suggestion
    default: './version.yml'
```

**Comment by rubenfiszel** [comment]
@mrchief Thanks for the review/feedback. The suggested changes make sense and were applied.

**Pull request merged**